### DISCUSSION TITLE
RichProgressBar is hard to read in light theme

### DISCUSSION THREAD

**Discussion by tshu-w** [Lightning-AI/pytorch-lightning#10833] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
As shown in the screenshot,RichProgressBar is hard to read in light theme.  I don't have much time to report a bug, so I open a discussion temporary.
<img width="1917" alt="Screen Shot 2021-11-30 at 10 47 40" src="https://user-images.githubusercontent.com/13161779/143977084-5182f702-6f41-4d39-85b3-1b202f5db309.png">

**Comment by rohitgr7** [comment]
cc @kaushikb11 @SeanNaren

**Comment by kaushikb11** [comment, marked as answer]
Thanks @tshu-w fr reporting the issue! I will look into if it's possible to detect theme in interactive environments.

Till then you could customize the `description` color by customizing `RichProgressTheme`

```python
trainer = Trainer(..., callbacks=RichProgressBar(theme=RichProgressBarTheme(description="black")))
```

Do check [RichProgressTheme](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/callbacks/progress/rich_progress.py#L166) out to customize styles for various components.

**Reply by tshu-w** [reply]
@kaushikb11 Sorry, I didn't read RichProgressBar arguments carefully.
